docker/nginx/etc/nginx.conf

```diff
@@ -0,0 +1,64 @@
+# /etc/nginx/nginx.conf
+user nginx;
+
+# Set number of worker processes automatically based on number of CPU cores.
+worker_processes auto;
+
+# Enables the use of JIT for regular expressions to speed-up their processing.
+pcre_jit on;
+
+# Configures default error logger.
+error_log /var/log/nginx/error.log notice;
+
+# Includes files with directives to load dynamic modules.
+include /etc/nginx/modules/*.conf;
+
+# Uncomment to include files with config snippets into the root context.
+# NOTE: This will be enabled by default in Alpine 3.15.
+include /etc/nginx/conf.d/*.conf;
+
+events {
+	worker_connections 1024;
+}
+
+http {
+    vhost_traffic_status_zone;
+
+    include /etc/nginx/mime.types;
+	default_type application/octet-stream;
+
+	server_tokens off;
+	client_max_body_size 100m;
+
+	sendfile on;
```

Comment:
inconsistent spacing - the block is idented with 3 spaces and here it's one tab. Please set it consistently, with other blocks too. Generally we tend to prefer 4 spaces.